packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/GitHub/index.tsx

```diff
@@ -10,7 +9,8 @@ import {
   Stack,
   Text,
 } from '@codesandbox/components';
-import { useAppState, useActions } from 'app/overmind';
+import { useGitHuPermissions } from 'app/hooks/useGitHubPermissions';
```

Comment:
For some reason `useGitHubPermissions` is `undefined` when I try to access it as an anonymous user:
```
Uncaught ReferenceError: useGitHubPermissions is not defined
```

Reply:
@tristandubbeld I guess the branch was out of date, I'm unable to reproduce it.

Reply:
Yes, it works now!